Source/WebCore/bindings/js/JSWorkerGlobalScopeBase.h

```diff
@@ -28,11 +28,8 @@
 
 #include "JSDOMGlobalObject.h"
 #include "JSDOMWrapper.h"
-#include "WorkerGlobalScope.h"
-
-#if ENABLE(SERVICE_WORKER)
 #include "ServiceWorkerGlobalScope.h"
-#endif
+#include "WorkerGlobalScope.h"
```

Comment:
Should it be ServiceWorkerGlobalScope.h instead?

Reply:
https://github.com/WebKit/WebKit/pull/20929